themes/root/templates/search/backtoorigin.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+/** @var \Laminas\View\Renderer\PhpRenderer $this */
+/** @var \VuFind\Search\SearchOrigin\AbstractSearchOrigin|null $searchOrigin */
+if (isset($searchOrigin)) {
+    $link = $this->url('alphabrowse-home') . '?' . http_build_query($searchOrigin->getOriginUrlParamsArray());
+    echo
+        '<div class="backtoorigin container">' .
+        '<p><a href="' . $link . '">' . $this->translate('Back to ' . $searchOrigin->getDisplayName()) . '</a></p>' .
```

Comment:
Unless I'm misunderstanding something, I think this may need to be updated to use the revised translation string.

Reply:
Also, is there a reason this is in the root theme rather than bootstrap3/bootstrap5? Generally the root theme is only used for things that are 100% style-free (e.g. templates for export transformation, etc.).

Reply:
I put it here as it's used by all the themes and the styling is theme dependant
But I can move it if you want to, should I?